pom.xml

```diff
@@ -35,7 +35,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <bullet.record.version>1.0.0</bullet.record.version>
+        <bullet.record.version>1.0.1</bullet.record.version>
```

Comment:
I am going to release that bullet-record as 1.1.0 since it breaks minor interfaces